usefulUtilities/statusIndicator/webApp/app/statusIndicator_webApp.js

```diff
@@ -374,9 +374,9 @@ function maybeCreateNewTables(response) {
     var query = `CREATE TABLE IF NOT EXISTS \`statusIndicator\` (
         username VARCHAR(100) PRIMARY KEY,
         displayName VARCHAR(100),
-        status VARCHAR(150),
+        status VARCHAR(150) DEFAULT 'busy',
         teamName VARCHAR(100) DEFAULT 'TBD',
-        location VARCHAR(100)
+        location VARCHAR(100) DEFAULT 'unknown'
```

Comment:
no code change necessary - but did you make sure this is reflected in the tables you already created? you can ensure that using phpmyadmin

Reply:
Not yet. I'll be "altering" table